Update README.md

agerwitz opened the pull request.

thibmaek commented on README.md:
```diff
@@ -1,5 +1,5 @@
-# pure-fun
-😊  A collection of Javascript pure functions/helpers with 0 dependencies
+# Pure-Fun
```
Why did you change this?

agerwitz commented:
I changed JavaScript because it is written with two capitals, "S" and "J."
Also, if one line is going to have a period at the end of the sentence,
then the other should. For the "pure-fun," I capitalized it because it is
the heading for the section. It usually stands out more and is more
visually appealing. I changed 0 to zero because numbers under 10 are
usually written out. But these are all dependent on preferences. I'm just
interested in grammar.

On Tue, Oct 3, 2017 at 9:50 AM, Thibault Maekelbergh <
notifications@github.com> wrote:

> *@thibmaek* requested changes on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/thibmaek/pure-fun/pull/6#discussion_r142407432>:
>
> > @@ -1,5 +1,5 @@
> -# pure-fun
> -😊  A collection of Javascript pure functions/helpers with 0 dependencies
> +# Pure-Fun
>
> Why did you change this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/thibmaek/pure-fun/pull/6#pullrequestreview-66762937>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AdAOxF86dqTFcXKpg33h-KlI2-ocAwmcks5sojutgaJpZM4PsISZ>
> .
>



-- 

AshleyGerwitz
Communication Analyst at Stark & Wayne
Phone: (716) 994-5470
Email: agerwitz@starkandw <agerwitz@starkandwayne.com>ayne.com
<agerwitz@starkandwayne.com>

thibmaek commented:
Other edits are fine but please don't capitalise pure-fun since it's a namespace rather than a title.

agerwitz commented:
Ok, sounds good. Sorry about that

On Tue, Oct 3, 2017 at 10:08 AM, Thibault Maekelbergh <
notifications@github.com> wrote:

> Other edits are fine but please don't capitalise pure-fun since it's a
> namespace rather than a title.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/thibmaek/pure-fun/pull/6#issuecomment-333853357>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AdAOxM8lYpfWSUeWOyC3OhNxe4rBB3R_ks5soj_TgaJpZM4PsISZ>
> .
>



-- 

AshleyGerwitz
Communication Analyst at Stark & Wayne
Phone: (716) 994-5470
Email: agerwitz@starkandw <agerwitz@starkandwayne.com>ayne.com
<agerwitz@starkandwayne.com>
